elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -60,7 +60,18 @@ done
 echo "List available plugins"
 ls -lrt $basedir
 
-# Download Knn lib 
+# Move RCA files
```

Comment:
Just realize is this an extra comment or you have more logic under this?

Reply:
It was meant to be a comment for the whole block. This is unnecessary so I just removed it.